WebColock.Tests/ClockControllerTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using WebClock.Controllers;
+using Xunit;
+using Newtonsoft.Json;
+using WebClock;
+
+namespace WebColock.Tests
+{
+    public class ClockControllerTests
+    {
+
+        private readonly ClockController _controller;
+
+        public ClockControllerTests()
+        {
+            _controller = new ClockController();
+        }
+
+        [Fact]
+        public void GetCurrentTime_WhenCalled_ReturnString()
+        {
+            var result = _controller.GetCurrentTime();
+            Assert.IsType<string>(result);
+        }
+
+        [Fact]
+        public void GetCurrentTime_WhenCalled_ReturnCorrectSecond()
+        {
+            var result = _controller.GetCurrentTime();
+
+            var currentDate= JsonConvert.DeserializeObject<Clock>(result);
+          
+                Assert.InRange(DateTime.Now.Second,currentDate.Second-15,currentDate.Second+15);
```

Comment:
Ten test raczej jest błędny. 
Pomyślcie co się stanie gdy currentDate to "14:50:59", a Now "14:51:00"? Moim zdaniem nie przejdzie, a powinno.
A co będzie gdy currentDate to "14:50:30", a Now "23:13:30"? Moim zdaniem test przejdzie, a chyba nie powinien.

Reply:
Racja. 
Teraz już powinno być dobrze.